packages/js/src/plugins/auctionHouseModule/operations/createBid.ts

```diff
@@ -335,8 +335,8 @@ export const createBidBuilder = async (
       })
 
       // Print the Bid Receipt.
-      .when(shouldPrintReceipt, (builder) =>
-        builder.add({
+      .when(shouldPrintReceipt, (builderArg) =>
+        builderArg.add({
```

Comment:
Why?

Reply:
shadowing. we have builder defined upper level